app/os_windows.go

```diff
@@ -1007,10 +1007,12 @@ func configForDPI(dpi int) unit.Metric {
 	}
 }
 
-// schemesDeeplink is a list of schemes, comma separated, that most be
+// schemesDeeplink is a list of schemes, comma separated, that must be
 // defined using -X compiler ldflag.
 var schemesDeeplink string
 var schemesDeeplinkList []string
+
+// startupDeeplink is the deeplink that started the app, if any.
 var startupDeeplink string
```

Comment:
Let's have all instances of `deeplink` replaced with `uri`, `openURI` or whatever makes sense. From the point of view of the Gio program, the feature is just about receiving URIs from outside.